### PR TITLE
Reduce diff with feature branch

### PR DESCRIPTION
**Pull request by pm47**
Best reviewed commit by commit. Those are minor changes that mostly touch tests.

### CODE REVIEW

File: eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala [hunk 1]
```diff
@@ -1699,7 +1699,7 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder, val
       }
       emitEvent_opt.foreach {
         case EmitLocalChannelUpdate(reason, d, sendToPeer) =>
-          log.info(s"emitting channel update event: reason=$reason enabled=${d.channelUpdate.channelFlags.isEnabled} sendToPeer=$sendToPeer realScid=${d.shortIds.real} channel_update={} channel_announcement={}", d.channelUpdate, d.channelAnnouncement.map(_ => "yes").getOrElse("no"))
+          log.debug(s"emitting channel update event: reason=$reason enabled=${d.channelUpdate.channelFlags.isEnabled} sendToPeer=$sendToPeer realScid=${d.shortIds.real} channel_update={} channel_announcement={}", d.channelUpdate, d.channelAnnouncement.map(_ => "yes").getOrElse("no"))
```
Comment (pm47) [inline eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala:1702]: @t-bast Is this okay or do we need that level of granularity for debug?

Alternatively I can filter out the "restore" reason, which is the spammy one

File: eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala [hunk 2]
```diff
@@ -134,15 +153,14 @@ class PeerSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with Paralle
     probe.send(peer, Peer.Connect(remoteNodeId, mockAddress_opt, probe.ref, isPersistent = true))
 
     // assert our mock server got an incoming connection (the client was spawned with the address from node_announcement)
-    awaitCond(mockServer.accept() != null, max = 30 seconds, interval = 1 second)
+    eventually(mockServer.accept() != null)
```
[the remark this answers is not in the document]
Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala:156]: 🤦‍♂️

File: eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala [hunk 3]
```diff
@@ -218,7 +239,7 @@ class PeerSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with Paralle
     val probe = TestProbe()
     switchboard.send(peer, Peer.Init(Set.empty))
 
-    awaitCond {
+    eventually {
       probe.send(peer, Peer.GetPeerInfo(None))
       probe.expectMsgType[Peer.PeerInfo].state == Peer.DISCONNECTED
```
Comment (t-bast) [inline eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala:244]:
```suggestion
      assert(probe.expectMsgType[Peer.PeerInfo].state == Peer.DISCONNECTED)
```

**Review by t-bast — COMMENTED** [review]
Last one and otherwise LGTM

File: eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala (hunk 3 above)
Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala:244]: Right! Went too fast on that one and assumed the `probe.expectMsgType[Peer.PeerInfo]` would throw

**Pull request merged**